helm/content-store/values.yaml

```
@@ -10,17 +10,21 @@ service:
   annotations: {}
   type: NodePort
   port: 80
-  targetPort: 80
-
 
 replicaCount: 1
 
-image:
+appImage:
   repository: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/content-store
   pullPolicy: IfNotPresent
   tag: latest
 
-containerPort: 80
+nginxImage:
+  repository: nginx
+  pullPolicy: IfNotPresent
+  tag: latest
```

I'd recommend using stable as a starting point. Latest can be the development branch with nginx: see their [supported tags](https://hub.docker.com/_/nginx). (Ultimately we might want to pin the versions and have something like Dependabot update them in a managed way, but that's way in the future.)
```suggestion
  tag: stable
```